site/en/docs/android/custom-tabs/guide-warmup-prefetch/index.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Warm-up and pre-fetching: using the Custom Tabs Service"
```

Comment:
I thought we had an easier way to do this with `android-browser-helper`, but can't find it (I thought maybe in the `TwaLauncher`). Long-term, an idea would to add that to that library.

Reply:
There's an easier way, but not enough to justify the overhead of explaining why to add another library.